tests/test_oml/test_functional/test_metrics/test_retrieval_metrics.py

```diff
@@ -162,3 +187,14 @@ def compare_metrics(
             values_expected = metrics_expected[metric_name][k]
             values_calculated = metrics_calculated[metric_name][k]
             assert torch.all(torch.isclose(values_expected, values_calculated, atol=1e-4)), [metric_name, k]
+
+
+def test_calc_fnmr_at_fmr(distances: torch.Tensor, mask_gt: torch.Tensor) -> None:
+    fmr_val = 0.1
+    fnmr_at_fmr = calc_fnmr_at_fmr(distances, mask_gt, fmr_val)
+    # positive distances are 0 0 1 1 2 2 5 5 9 9
```

Comment:
I guess this can be moved (or copied) to the original metric function `calc_fnmr_at_fmr`